src/components/SearchResults.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "../styles/search-results.css";
+
+const SearchResults = ({ searchResults }) => {
+    console.log(searchResults, "renders search results")
+    if (!searchResults.length) {
+        return <p className="no-results">Sorry, no results found. Try again. </p>
+    } else {
+    return (
+        <>
+        <div className="search-results">
+            {searchResults.map((image) => {
+                console.log(image, "image")
+            return <img className="search-results__image"
+                    src={image}
+                    alt="spaceImage" />
+            })}
+        </div>
+        </>
+        );
+    }
+}
+
```

Comment:
Should only ever have one blank line clear anywhere for spacing, one of these should be removed